src/main/java/org/osiam/resources/exception/NoSuchElementException.java

```diff
@@ -0,0 +1,31 @@
+/*
```

Comment:
It's not recognized by github, but in git the history is available. So ignore my comment.